packages/@okta/vuepress-site/docs/guides/implement-saml2/configure-policy-as/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Configure the Authorization Server policy
+---
+
+To enable consent for the [scopes](/docs/reference/api/authorization-servers/#create-a-scope) that you want to require consent for, select **API**, **Authorization Servers**, and then select **default** (Custom Authorization Server) in the table. In this example, we are enabling consent for default Custom Authorization Server scopes.
+
+Make sure that the SAML 2.0 Assertion grant type is enabled in the Authorization Server default policy rule.
```

Comment:
it is not necessary to be in the default policy rule, the admin can either change an existing rule or create a new rule